### PR TITLE
🔢 Change Homepage button based on user

### LINKED ISSUE #348: Dynamically Changing Homepage Button Based on User
#### Description
Currently, the homepage displays find a Mentor even if you yourself a logged in as a Mentor which is confusing. 
- Change this so that if you are a mentor it directs you to the Mentee gallery
- For admin, you can default to Mentor Gallery 

#### Screenshots
image

#### Files
`Home.js`

### PR DESCRIPTION
**Pull request by zfaizal2**
## Description :star2:

Fixes #348 
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@
 venv
 node_modules
 backend/firebase_service_key.json
-package-lock.json
\ No newline at end of file
+package-lock.json
+package.json
```
Comment (Leundai) [inline .gitignore:6]: we should track this just in case there are some changes. This won't be automatically changed by yarn

**Review by Leundai — CHANGES_REQUESTED** [review]
lookin good

**Review by pguduguntla — COMMENTED** [review]
Looks good to me.

File: frontend/src/components/pages/Home.js [hunk 2]
```diff
@@ -14,11 +16,15 @@ function Home({ history }) {
...
           <br />
           <LoginVerificationModal
-            content={<b>Find a Mentor</b>}
+            content={isMentor ? <b>Find a Mentee</b> : <b>Find a Mentor</b>}
             theme="dark"
             onVerified={() => {
+              let redirect = "/gallery";
+              if (isMentor) {
+                redirect = "/mentee-gallery";
+              }
```
Comment (Leundai) [inline frontend/src/components/pages/Home.js:25]: A super small thing, could you add these as a const to the file `consts.js`

**Review by Leundai — APPROVED** [review]
looks good!

**Review by pguduguntla — APPROVED** [review]
Looks good.

**Pull request merged**